pkg/apis/toolchain/v1alpha1/nstemplatetier_types.go

```diff
@@ -16,6 +16,9 @@ type NSTemplateTierSpec struct {
 	// The namespace templates
 	// +listType
 	Namespaces []NSTemplateTierNamespace `json:"namespaces"`
+
+	// the cluster resources template (for cluster-wide quotas, etc.)
+	ClusterResources NSTemplateTierClusterResources `json:"clusterResources"`
```

Comment:
Let's make it optional.
```suggestion
	// the cluster resources template (for cluster-wide quotas, etc.)
	// +optional
	ClusterResources NSTemplateTierClusterResources `json:"clusterResources,omitempty"`
```